pkg/ucp/frontend/controller/radius/proxy.go

```diff
@@ -220,8 +245,8 @@ func (p *ProxyController) IsTerminalResponse(resp *http.Response) bool {
 }
 
 // UpdateTrackedResource updates the tracked resource synchronously.
-func (p *ProxyController) UpdateTrackedResource(ctx context.Context, downstream string, id resources.ID, apiVersion string) error {
-	return p.updater.Update(ctx, downstream, id, apiVersion)
+func (p *ProxyController) UpdateTrackedResource(ctx context.Context, downstreamURL string, originalID resources.ID, apiVersion string) error {
+	return p.updater.Update(ctx, downstreamURL, originalID, apiVersion)
 }
```

Comment:
why are we calling  ID as originalID? are we modifying it somewhere?

Reply:
The tracked resource code has to manage a few different kinds of resource IDs. The `originalID` refers to the resource being processed rather than one of the derived values. It's not new in this PR.